framework/doc/content/source/actioncomponents/ComponentBoundaryConditionInterface.md

```diff
@@ -0,0 +1,17 @@
+# ComponentBoundaryConditionInterface
+
+The `ComponentBoundaryConditionInterface` is a base class designed to facilitate the use of [Physics](Physics/index.md)
+by an [ActionComponent.md]. It offers:
+
+- the [!param](/ActionComponents/CylinderComponent/fixed_value_bc_variables), [!param](/ActionComponents/CylinderComponent/fixed_value_bc_boundaries) and [!param](/ActionComponents/CylinderComponent/fixed_value_bc_values) to define fixed value / Dirichlet boundary conditions on the surfaces of the `ActionComponent`
+- the [!param](/ActionComponents/CylinderComponent/flux_bc_variables), [!param](/ActionComponents/CylinderComponent/flux_bc_boundaries) and [!param](/ActionComponents/CylinderComponent/flux_bc_values) to define flux boundary conditions on the surfaces of the `ActionComponent`
+
+An [ActionComponent.md] inheriting `ComponentBoundaryConditionInterface` must be registered to the `init_component_physics` and `check_integrity`
```

Comment:
Optional: do we have any way to check/enforce this?

Reply:
we do in Physics with the addRequiredTask routine

Reply:
and in ActionComponent
```
  /// Add a new required task for all physics deriving from this class
  /// NOTE: This does not register the task, you still need to call registerMooseAction
  void addRequiredTask(const std::string & task) { _required_tasks.insert(task); }
```

I ll make sure it's used

Reply:
done